config/octane.php

```diff
@@ -218,4 +218,19 @@
 
     'max_execution_time' => 30,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Status Code Reasons for Octane Swoole
+    |--------------------------------------------------------------------------
+    |
+    | If you use non-standard HTTP status codes in your response, Octane gonna
```

Comment:
Can you apply the style of "3 characters less than the previous line" in this comment please?